--- libopensync-0.31/SConstruct~	2007-08-31 21:31:59.000000000 +0300
+++ libopensync-0.31/SConstruct	2007-08-31 21:37:49.797518617 +0300
@@ -35,8 +35,8 @@
 A(BoolOption('enable_python', 'Build python wrapper? (swig required)', 0))
 A(BoolOption('debug_modules', 'Should unloading of shared modules be avoided (DEBUGGING ONLY!)', 0))
 A(BoolOption('enable_doxygen', 'Generating OpenSync API with doxygen?', 0))
-A(('APPEND_LDFLAGS', 'Linker flags'))
-A(('APPEND_CCFLAGS', 'Compiler flags'))
+A(('LDFLAGS', 'Linker flags'))
+A(('CCFLAGS', 'Compiler flags'))
 A(('DESTDIR', 'Set the root directory to install into ( /path/to/DESTDIR )', ''))
 
 target_dir = SelectBuildDir('build')
@@ -58,9 +58,6 @@
 You can set the following options:
 """ + env.get_opts().generate_help_text())
 
-env.Append(CCFLAGS = Split('$APPEND_CCFLAGS'))
-env.Append(LDFLAGS = Split('$APPEND_LDFLAGS'))
-
 env.add_define("OPENSYNC_PLUGINDIR", config['plugindir'])
 env.add_define("OPENSYNC_FORMATSDIR", config['formatdir'])
 env.add_define("OPENSYNC_CONFIGDIR", config['configdir'])
@@ -75,7 +72,9 @@
 
 env.Replace(
 	CC = env['CC'],
-	CXX = env['CXX']
+	CXX = env['CXX'],
+	CXXFLAGS = env['CXXFLAGS'],
+	CFLAGS = env['CFLAGS'],
 )
 
 if env['enable_doxygen'] == 1:
