--- ./wrapper/opensync-group.i.org	2007-08-31 20:12:48.579835095 +0200
+++ ./wrapper/opensync-group.i	2007-08-31 20:13:37.396480039 +0200
@@ -28,7 +28,10 @@
 	}
 
 	void add_group(Group *group) {
-		osync_group_env_add_group(self, group);
+                Error *err = NULL; 
+                bool ret = osync_group_env_add_group(self, group, &err); 
+                if (!raise_exception_on_error(err) && !ret) 
+                        wrapper_exception("osync_group_env_add_group failed but did not set error code"); 
 	}
 
 	void remove_group(Group *group) {
